resolwe/flow/executors/docker/run.py

```diff
@@ -124,6 +124,8 @@ def _processing_volumes(self) -> Dict:
         base_path = Path(SETTINGS["FLOW_EXECUTOR"].get("DATA_DIR", ""))
         local_data = base_path / f"{storage_url}_work"
         local_data.mkdir()
+        # Make sure that tmp dir exists.
```

Comment:
The commit message here is misleading, because tmp dir was already inside the data volume.

Reply:
I have removed TMPDIR creation and fixed the commit message.